src/cmd/serial.js

```diff
@@ -211,7 +225,31 @@ module.exports = class SerialCommand extends CLICommandBase {
 		}
 
 		function reconnect(){
```

Comment:
@monkbroc Could you kindly take a look at this refactor and let me know if you see this is not ok for merge?

Once the `serial monitor --follow` command is reconnecting to the device, the usb handle is unable to be obtained via `usbUtils.getOneUsbDevice` quickly enough so I added an arbitrary delay. Tested on Argon and it takes a few milliseconds (have not tested yet on P2 coz unfortunately my P2 is at work location and I will get it later today). Picked 10 ms without solid reasoning.